Doc/library/stdtypes.rst

```diff
@@ -1549,6 +1549,39 @@ expression support in the :mod:`re` module).
    interpreted as in slice notation.
 
 
+.. method:: str.removeprefix(prefix, /)
+
+   Return a copy of the string with the given prefix removed, if present. ::
+
+      >>> 'BarFooBaz'.removeprefix('Bar')
+      'FooBaz'
+      >>> 'BarFooBaz'.removeprefix('Baz')
+      'BarFooBaz'
```

Comment:
These examples are not easy to read since Bar and Baz look the same. I suggest to use different strings:

```suggestion
      >>> 'TestHook'.removeprefix('Test')
      'Hook'
      >>> 'BaseTestCase'.removeprefix('Test')
      'BaseTestCase'
```